data/hai/hai ships.txt

```diff
@@ -555,6 +555,20 @@ ship "Lightning Bug" "Lightning Bug (Unfettered Shipyards)"
 		`"Biroo" Atomic Steering`
 		"Hyperdrive"
 
+ship "Lightning Bug" "Lighting Bug (Tripulse)"
```

Comment:
```suggestion
ship "Lightning Bug" "Lightning Bug (Tripulse)"
```

Reply:
Can't you add these changes yourself? This is just a typo.